### ISSUE TITLE
Render media files

### ISSUE THREAD

**Issue by PatMyron** [pomber/git-history#61]
It would be awesome to traverse media file history. Currently, they display raw binary data:

https://githistory.xyz/PatMyron/patmyron.github.io/blob/master/pics/football.png
https://githistory.xyz/PatMyron/patmyron.github.io/blob/master/resume.pdf

**Comment by pomber** [comment]
Of course, I forgot about media files. I'd love to add this eventually (and maybe with [cool effects](https://tympanus.net/Tutorials/PiecesSlider/)).
